src/data/publications/publications.yaml

```diff
@@ -1,3 +1,23 @@
+-
+  title: "Sequencing in over 50,000 cases identifies coding and structural variation underlying atrial fibrillation risk"
+  date: 2025-03-06
+  location: Nature Genetics
+  url: https://www.nature.com/articles/s41588-025-02074-9
+-
+  title: "Best Practices for Clinical Trials Data Harmonization and Sharing on NHLBI BioData Catalyst (BDC) Learned from CONNECTS Network COVID-19 Studies"
+  date: 2025-03-26
+  location: Journal of Clinical and Translational Science
+  url: https://scholar.google.com/scholar_url?url=https://www.cambridge.org/core/services/aop-cambridge-core/content/view/790583F6D473634FD94995324A361752/S2059866125000524a.pdf/best-practices-for-clinical-trials-data-harmonization-and-sharing-on-nhlbi-biodata-catalyst-bdc-learned-from-connects-network-covid-19-studies.pdf&hl=en&sa=X&d=4855232722376886784&ei=-QvoZ-ilIOOO6rQPh_m8qQ8&scisig=AFWwaeZvdxlncouSA_epU9q-d29Z&oi=scholaralrt&hist=XoZNe50AAAAJ:13037493336686039525:AFWwaebyIBBV7PZoY3977yv4blKM&html=&pos=0&folt=kw-top
```

Comment:
probably just use the redirected url: https://www.cambridge.org/core/services/aop-cambridge-core/content/view/790583F6D473634FD94995324A361752/S2059866125000524a.pdf/best-practices-for-clinical-trials-data-harmonization-and-sharing-on-nhlbi-biodata-catalyst-bdc-learned-from-connects-network-covid-19-studies.pdf

Reply:
good catch! thanks for finding the cleaner URL!